pom.xml

```diff
@@ -220,21 +190,37 @@
 				</executions>
 			</plugin>
 			<plugin>
-				<groupId>org.jetbrains.dokka</groupId>
-				<artifactId>dokka-maven-plugin</artifactId>
-				<version>${dokka-maven-plugin.version}</version>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-invoker-plugin</artifactId>
+				<version>${maven-invoker-plugin.version}</version>
+				<dependencies>
+					<dependency>
+						<groupId>org.apache.groovy</groupId>
+						<artifactId>groovy</artifactId>
+						<version>${maven-invoker-plugin.groovy.version}</version>
+					</dependency>
+				</dependencies>
+				<configuration>
+					<projectsDirectory>src/test/it</projectsDirectory>
+					<showErrors>true</showErrors>
+					<cloneProjectsTo>${project.build.directory}/its</cloneProjectsTo>
+					<pomIncludes>
+						<pomInclude>*/pom.xml</pomInclude>
+					</pomIncludes>
+					<settingsFile>src/test/it/settings.xml</settingsFile>
+					<localRepositoryPath>${project.build.directory}/it-repo</localRepositoryPath>
+					<postBuildHookScript>verify.groovy</postBuildHookScript>
```

Comment:
Any chance to do the tests in Kotlin instead? (e.g. Kotlin Scripting)

Or is this some hardcoded thing in the invoker plugin?